docs/release-notes/2.26.0.md

```diff
@@ -0,0 +1,90 @@
+* [PMM-9159](https://jira.percona.com/browse/PMM-9159): DBaaS: PMM now supports Percona Kubernetes Operator for Percona XtraDB Cluster 1.10.0.
+* [PMM-9180](https://jira.percona.com/browse/PMM-9180): Integrated Alerting:  Add Alert Rule: Added the Template expression in a collapsible panel for an enhanced user experience (default view as collapsed) as the technical message could confuse the users.
+* [PMM-7781](https://jira.percona.com/browse/PMM-7781): Integrated Alerting: Alert rules no longer depend on their source rule template after creation. This means that you can now update or delete rule templates without impacting existing rules that are based on that template. For more information, see [Integrated Alerting](../using/alerting.md).
+* [PMM-9356](https://jira.percona.com/browse/PMM-9356): Added new experimental Environment dashboards in PMM.
+* [PMM-9296](https://jira.percona.com/browse/PMM-9296): Disclaimer about Technical Preview feature added to Percona Platform - Connect PMM to Percona portal page.
+
+
+## Bugs Fixed
+
+* [PMM-8867](https://jira.percona.com/browse/PMM-8867): Fixed an issue for PMM Client installation using the tarball script (without using RPMs) where the configuration was getting lost due to the configuration file (pmm-agent.yml) being recreated.
+* [PMM-8094](https://jira.percona.com/browse/PMM-8094): DBaaS: Fixed an issue for paused clusters that froze with PSMDB v1.8 operators when all the pods were terminated, providing no cluster resumption option.
+* [PMM-8535](https://jira.percona.com/browse/PMM-8535): DBaaS: Repeating error after force unregister (Thanks to Beata Handzelova for reporting this issue)
+* [PMM-9144](https://jira.percona.com/browse/PMM-9144): Dashboard: Fixed the Add inventory page issue that indicated AWS RDS/Aurora supported only MySQL.
+* [PMM-9289](https://jira.percona.com/browse/PMM-9289): "Get from Browser" on settings page does not fetch port in Public address field breaking the integration for Platform authentication.
+* [PMM-9255](https://jira.percona.com/browse/PMM-9255): On connecting the PMM server to Percona Platform for an admin user insufficient access rights error message is thrown.
+* [PMM-9049](https://jira.percona.com/browse/PMM-9049): Eliminated confusion around the current and available version date by adding a tooltip with an explanation for these dates.
+* [PMM-9181](https://jira.percona.com/browse/PMM-9181): Integrated Alerting: Modified the label for the enable/disable button in order to avoid confusion.
+* [PMM-5405](https://jira.percona.com/browse/PMM-5405): Fixed an issue where the `pmm-admin summary` command fails if a null value is passed for the `--filename` parameter.
+* [PMM-8141](https://jira.percona.com/browse/PMM-8141): Fixed an issue for VictoriaMetrics not capturing the metrics as the configuration file generated by the PMM agent was absent.
```

Comment:
It's unclear user impact here, as we do not expect regular users to know about VictoriaMetrics. 
so this can be something like " fixed an issue when no metrics captured if temporary folder on Client node cleaned. This was causing removal of configuration file on Clent side "  (if I understand the problem correctly)